media/src/App.svelte

```diff
@@ -328,21 +329,12 @@
     const makeQueryStringObject = function() {
         const flattenedObjects = {
             currentChapter,
-            shadeUp,
-            flipInfo,
-            grid: gridMeshes.visible,
         };
-        objects.forEach((object, index) => {
-            const prefix = `obj${index}_`;
-            flattenedObjects[prefix + "kind"] = object.kind;
-            if (object.params) {
-                for (const [key, value] of Object.entries(object.params)) {
-                    flattenedObjects[prefix + "params_" + key] = value;
-                }
-            }
-        });
-        const urlParams = new URLSearchParams(flattenedObjects);
-        window.location.search = urlParams.toString();
+        if (flipInfo) flattenedObjects['flipInfo'] = true;
```

Comment:
Just a minor note, we've generally always used the longhand form for if statements like this - I prefer it as it makes the code more clear. It's really not a big deal and up to personal preference though, so it's fine to leave as is here.
```
if (flipInfo) {
    flattenedObjects['flipInfo'] = true;
}
```